src/components/command-bar/commands/search/components/no-results-message/index.tsx

```diff
@@ -4,14 +4,14 @@
  */
 
 import { ReactElement } from 'react'
-import { SearchableContentType } from 'contexts'
+import { CurrentContentType } from 'contexts'
 import Text from 'components/text'
 import { useCommandBar } from 'components/command-bar'
 import s from './no-results-message.module.css'
 
 interface NoResultsMessageProps {
 	tabsWithResults: {
-		type: SearchableContentType
+		type: CurrentContentType
```

Comment:
Updated this as the `NoResultsMessage` now needs to support the `global` content type, which is part of the `CurrentContentType` but not `SearchableContentType`.